### ISSUE TITLE
a11y improvements for CopyLink

### ISSUE THREAD

**Issue by maria-jon** [openaq/openaq.org#1123]
CopyLink accessibility can improve in a few places: 

## CopyLink

- “Copy link” button SVG missing alts/aria-hidden  
- Links missing text label & SVG alts/aria-hidden
